stable/traefik/values.yaml

```diff
@@ -30,6 +30,8 @@ resources: {}
 debug:
   enabled: false
 
+logLevel: error
```

Comment:
Can you have this disabled by default (commented out)?

Reply:
Isn't this the default? Would it be better if I set this to `info` as per your previous comment?

Reply:
@dtomcej am I correct here and this is the default or do I need to put an `if` statement in?

Thanks,
Paul

Reply:
There is actually a default that you can put in the template, which works well for this instance!

https://helm.sh/docs/chart_template_guide/#using-the-default-function

Reply:
ah brilliant, learn something new every day. I have made the requested changes